packages/server/postgres/migrations/1627394777775_addPollsTable.ts

```diff
@@ -0,0 +1,58 @@
+import {MigrationBuilder, ColumnDefinitions} from 'node-pg-migrate'
+import {Client} from 'pg'
+
+import getPgConfig from '../getPgConfig'
+
+export const shorthands: ColumnDefinitions | undefined = undefined
+
+export async function up(pgm: MigrationBuilder): Promise<void> {
+  const client = new Client(getPgConfig())
+  await client.connect()
+
+  await client.query(`
+    DO $$
+    BEGIN
+      IF NOT EXISTS (SELECT 1 FROM pg_type WHERE typname = 'PollStatusEnum') THEN
+        CREATE TYPE "PollStatusEnum" AS ENUM (
+          'creating',
+          'active',
+          'editing',
+          'ended'
+        );
+      END IF;
+    END
+    $$;
+
+    CREATE TABLE IF NOT EXISTS "Poll" (
+      "id" VARCHAR(100) PRIMARY KEY NOT NULL,
+      "createdAt" TIMESTAMP WITH TIME ZONE DEFAULT CURRENT_TIMESTAMP NOT NULL,
+      "updatedAt" TIMESTAMP WITH TIME ZONE DEFAULT CURRENT_TIMESTAMP NOT NULL,
+      "deletedAt" TIMESTAMP WITH TIME ZONE DEFAULT NULL, 
+      "createdById" VARCHAR(100) NOT NULL REFERENCES "User"("id"),
+      "discussionId" VARCHAR(100) NOT NULL REFERENCES "Discussion"("id"),
+      "winningOptionId" VARCHAR(100),
+      "title" VARCHAR(3000),
```

Comment:
this feels a little long. maybe 300? I'm AOK with the DB limits being bigger than application limits, but i can't imagine application limits being much more than ~200 or so chars

Reply:
Good catch! I'll lower both. Poll title to 300 and the poll option title to 100 (designs show it can't be longer than 30, but just to keep it safe) 👍